tests/openvino/test_training.py

```diff
@@ -98,6 +100,11 @@ def initialize_movement_sparsifier_parameters_by_sparsity(
                 operand.bias_importance.copy_(bias_init_tensor)
 
 
+def is_avx512_vnni_supported() -> bool:
+    flags = [flag.lower() for flag in cpuinfo.get_cpu_info()["flags"]]
+    return "avx512_vnni" in flags or "avx512vnni" in flags
```

Comment:
```suggestion
    return "avx512_vnni" in flags or "avx512vnni" in flags or "avx_vnni" in flags
```

Alder Lake and Raptor Lake do not support avx512 but do have VNNI. When I checked on a model with overflow_fix enabled accuracy was fine there.

Reply:
@helena-intel that is a good observation. Client CPUs provide different VNNI setup. I think the keyword should be just "vnni".

Reply:
i have added the "avx_vnni" flag. Since this is intended for tests, i think "avx512_vnni", avx512vnni, avx_vnni should be able to cover

Reply:
Changed to `re.search("avx.*vnni", flag) is not None` to capture possible flags, e.g., avx512_vnni, avx2_vnni, avx_vnni, avx_vnni_int8, x86_avx_vnni